PureBasicIDE/CompilerInterface.pb

```diff
@@ -77,7 +77,7 @@ CompilerIf #CompileWindows
     #COMPILER_UNICODE    = " /UNICODE" ; still needed to set in older compilers
   CompilerEndIf
   
-  #COMPILER_FileFormat = #PB_UTF8
+  #COMPILER_FileFormat = #PB_Ascii
```

Comment:
This shouldn't be changed. The Windows compiler is now unicode and communicate using UTF-8